### PR TITLE
Add setting to use editor for WooCommerce emails [MAILPOET-2284]

### PR DESCRIPTION
**Pull request by NeosinneR**
[MAILPOET-2284]

[MAILPOET-2284]: https://mailpoet.atlassian.net/browse/MAILPOET-2284

### CODE REVIEW

File: views/settings/woocommerce.html [hunk 1]
```diff
@@ -22,6 +22,11 @@
         <a class="button-secondary mailpoet_woocommerce_editor_button" href="?page=mailpoet-newsletter-editor&id=<%= settings.woocommerce.transactional_email_id %>"><%= _x('Open template editor', 'Settings button to go to WooCommerce email editor') %></a>
         <a class="button-secondary mailpoet_woocommerce_editor_button_disabled" disabled="disabled"><%= _x('Open template editor', 'Settings button to go to WooCommerce email editor') %></a>
       </p>
+      <input
+        type="hidden"
+        value="<%= settings.woocommerce.transactional_email_id %>"
+        name="woocommerce[transactional_email_id]"
+      >
```
Comment (amine-mp) [inline views/settings/woocommerce.html]: Nice catch, I did think about that when I added the setting!

**Comment by codemonkey-jack** [comment]
@amine-mp I have the transactional email ID in my database;

image

However, I don't see this new setting in `wp-admin > MailPoet > Settings > WooCommerce`. 

Can you please clarify if anything specific needs to be done to make it work?

**Comment by NeosinneR** [comment]
@codemonkey-jack do you have feature flag turned on?

**Comment by codemonkey-jack** [comment]
@NeosinneR Yep :-/ the `wc-transactional-email` feature flag. Maybe I have the wrong version of the premium? I'm just using the latest master as there was no PR linked to premium for this PR.

**Pull request merged**